systers_portal/templates/account/signup.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+
+{% load account %}
+
+{% block title %} - Sign Up{% endblock %}
+
+{% load crispy_forms_tags %}
+{% block content %}
+  <div class="container">
+    <form class="login form-signin" id="signup_form" method="post" action="{% url 'account_signup' %}">
```

Comment:
I see you are reusing `form-signin` class. Could you please create a new one `form-signup` class in the CSS file and keep same styling. Just in case we ever want to change any of the classes and not break other forms accidentally. Something like that:

```CSS
form-singin, form-singup {
  // same old styling here
}